pkg/config/generate.go

```diff
@@ -211,6 +211,7 @@ func Generate(cs *acsapi.OpenShiftManagedCluster) (err error) {
 		// Openshift Console is BYO type of certificate. In the long run we should
 		// enable users to configure their own certificates.
 		// For this reason we decouple it from all OCP certs and make it self-sign
+		// If FQDN matches PublicHostname certificate can't be self-sign
```

Comment:
bit more detail here would be good (or link to issue)

Reply:
sorry, meant link to https://github.com/openshift/openshift-azure/issues/307

Reply:
Its expected behavior of named_certificates for the console. The issue is a consequence of me failing to implement it the first time. I think the issue will become irrelevant soon?